src/widgets/block.rs

```diff
@@ -191,24 +299,30 @@ impl<'a> Block<'a> {
         self
     }
 
-    /// Applies the style to all titles. If a title already has a style, it will add on top of it.
+    /// Applies the style to all titles.
```

Comment:
All titles have a style - even when that style is `Style::default()` It's worth talking about how the combination of a default style, plus the specific override is done using consistent language **across all of the docs**. Consistency on this is notable because the order of applying styles has come up in discord / issues before as a point of confusion, and we use this pattern a lot.

What about something like:
"Sets the default style for titles which may be overridden by the styles on each individual title"

I'm not quite sure that "override" is quite the right verb here as the styles are combined, not replaced, and the default is applied first.

Thoughts?